spylib/webhook.py

```diff
@@ -0,0 +1,20 @@
+from enum import Enum
+from pathlib import Path
+
+from pydantic import BaseModel
+
+WEBHOOK_CREATE_GQL = Path('./spylib/webhook.graphql').read_text()
```

Comment:
I assume the `.graphql` won't get stripped out when deploying to something like an aws lambda?

Reply:
Did we encounter some problem like that before? 

> I assume the `.graphql` won't get stripped out when deploying to something like an aws lambda?

Reply:
Alternatively, we can make it a string.

Reply:
I have never seen this as a problem. Let's keep like this until we have a reason to change it back to string.